src/node/util.ts

```diff
@@ -221,6 +221,28 @@ export function isArray(a: unknown): boolean {
   return Array.isArray(a);
 }
 
+export function parseEnv(): void {
+  // We don't implement this function yet because it is currently under
+  // active development. We should implement it when it's stable
+  throw new Error('node:util parseEnv is not implemented');
+}
+
+export function getSystemErrorMap(): void {
+  throw new Error('node:util getSystemErrorMap is not implemented');
+}
+
+export function getSystemErrorName(): void {
+  throw new Error('node:util getSystemErrorName is not implemented');
+}
+
+export function getSystemErrorMessage(): void {
+  throw new Error('node:util getSystemErrorMessage is not implemented');
+}
+
+export function styleText(): void {
+  throw new Error('node:util styleText is not implemented');
```

Comment:
Just noting... there's really no reason not to implement this one... it's not likely super useful in the common workers case but there's also no other reason not to implement it.